cmake/fw_zip.cmake

```diff
@@ -30,7 +29,7 @@ function(generate_dfu_zip)
     "type=${GENZIP_TYPE}"
     "board=${CONFIG_BOARD}"
     "soc=${CONFIG_SOC}"
-    DEPENDS ${GENZIP_TARGET}
```

Comment:
Removing this target dependency is generally dangerous.

Discussion with @tejlmand is ongoing to determine if this is safe in this case.